ShadowDOM/explainer.md

```diff
@@ -243,6 +245,20 @@ Another advantage of this proposal is that it can allow multiple module specifie
 </my-element>
 ```
 
+### Scoping
+
+The module map exists today as a global registry per document, not scoped to a particular shadow root. Many developers have expressed interest in such a global map for sharing stylesheets, as it allows for nested shadow roots to access a base set of shared styles without needing to redefine them at each level of shadow root nesting.
+
+A global map does come with some tradeoffs, particularly when names collide. With a global map, nested shadow roots could override entries from parent shadow roots, which could be undesirable.
+
+### `<script>` vs `<style>` For CSS Modules
```

Comment:
I wonder if we should show a code example of what using `<style>` would look like?

I also wonder if we could note that both options could be supported, as well (not sure if that is something we want to propose or not, but having two ways to do things seems reasonable as well, if it makes sense)

Reply:
Great suggestions, updated with both an example of `<style>` as well as proposing supporting both. I'm not against both, but it could be controversial, as there are guidelines about redundancy and competing guidelines about consistency.